lib/3scale/backend/metric.rb

```diff
@@ -162,6 +162,17 @@ def descendants(service_id, metric_name)
           end
         end
 
+        # Returns the "ascendants" of a metric, that is, its parents,
```

Comment:
can a metric have multiple parents?

Reply:
No.

Reply:
yep, this was meant to correct the parents to `parent, grandparent, ...`

Reply:
Right. The description is not accurate :+1:

Reply:
Fixed :+1: